include/tinycompress/compress_ops.h

```diff
@@ -0,0 +1,40 @@
+/* SPDX-License-Identifier: (LGPL-2.1-only OR BSD-3-Clause) */
+/* Copyright (c) 2020 The Linux Foundation. All rights reserved. */
+
+#ifndef __COMPRESS_OPS_H__
+#define __COMPRESS_OPS_H__
+
+#include "sound/compress_params.h"
+#include "sound/compress_offload.h"
+#include "tinycompress.h"
+
+struct compress_ops {
```

Comment:
Can we add documentation for these

Reply:
tinycompress.h already has documentation per API. Do I just add generic comment for struct compress_ops or have comment for each API again ?